Assets/Board.cs

```diff
@@ -8,14 +9,17 @@ public class Board : MonoBehaviour {
     public int boardWidth = 20;
     public Transform piecePrefab;
     public Transform borderPrefab;
+    public Transform placePrefab;
     public Text score;
     public Slider timer;
     BoardModel board = new BoardModel();
     
     Piece current;
     List<Transform> shadow = new List<Transform>();
     List<Vector2> validPositions = new List<Vector2>();
+    HashSet<PlaceView> selectedPlaces = new HashSet<PlaceView>();
     Dictionary<Pos,Transform> transforms = new Dictionary<Pos,Transform>();
+    Dictionary<Pos,PlaceView> places = new Dictionary<Pos,PlaceView>();
```

Comment:
I should stop with those Dictionary of position... a good old grid will a better job